lib/internal/inspector/network_undici.js

```diff
@@ -91,6 +93,21 @@ function onClientResponseHeaders({ request, response }) {
   if (typeof request[kInspectorRequestId] !== 'string') {
     return;
   }
+
+  let mimeType;
+  let charset;
+  try {
+    const contentTypeKeyIndex =
+      ArrayPrototypeFindIndex(response.headers, (header) => header.toString().toLowerCase() === 'content-type');
+    const contentType = contentTypeKeyIndex !== -1 ? response.headers[contentTypeKeyIndex + 1].toString() : '';
+    const mimeTypeObj = new MIMEType(contentType);
+    mimeType = mimeTypeObj.essence || '';
+    charset = mimeTypeObj.params.get('charset');
```

Comment:
Charset can be undefined/null if this does not exist. I don't think we want that.

Reply:
I overlooked this case.
Like mimeType, I’ve set it to an empty string.
Or should we avoid supporting charset at all?